00-open-a-shapefile.Rmd

```diff
@@ -136,19 +134,24 @@ shapefiles include
 
 * a polygon shapefile representing our study area boundary, 
 * a line shapefile representing roads and streams, and 
-* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) located at the NEON 
-Harvard Forest field site.
+* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) 
+located at the NEON Harvard Forest field site.
 
```

Comment:
Should the linking text for the flux tower collection methods be changed to this format as well?